roles/Cargo.toml

```diff
@@ -23,6 +23,7 @@ members = [
     "translator",
     "jd-client",
     "jd-server",
+    "tests-integration",
```

Comment:
might be misleading? or can be `roles-integration-tests`

Reply:
Yeah I also prefer `roles-integration-tests`. Also in the relative `Cargo.toml`

Reply:
I think it should just be `tests`.

Reply:
initially  `tests-integration` was chosen because there is already a `tests-utils` folder and I wanted to be consistent. Is it worth deciding on the folder naming now, or when we know better how the role's crates are going to be structured/named?  I prefer to keep it as is and rename it when restructuring the `roles` crates is underway, otherwise happy to change to either `tests` or `roles-integration-tests`, whatever gets more thumbs up.

Reply:
makes sense!!!